server/src/main/java/org/apache/uniffle/server/ShuffleServerConf.java

```diff
@@ -352,9 +352,7 @@ public class ShuffleServerConf extends RssBaseConf {
   public static final ConfigOption<Long> FALLBACK_MAX_FAIL_TIMES =
       ConfigOptions.key("rss.server.hybrid.storage.fallback.max.fail.times")
           .longType()
-          .checkValue(
-              ConfigUtils.NON_NEGATIVE_LONG_VALIDATOR, " fallback times must be non-negative")
-          .defaultValue(0L)
+          .defaultValue(-1L)
```

Comment:
1. What if the users set it to 0.
2. If `!storageManager.canWrite(flushEvent)`, we should ignore retry times. So should we add a new argument named `force` to `org.apache.uniffle.server.storage.AbstractStorageManagerFallbackStrategy#tryFallback` for ignoring retry times. WDYT? @zuston  @jerqi 

https://github.com/apache/incubator-uniffle/blob/318050144b62c0256c151a278bb04ac73c6e544b/server/src/main/java/org/apache/uniffle/server/storage/hybrid/FallbackBasedStorageManagerSelector.java#L49-L51

Reply:
SGTM.

Reply:
Ok for me.